examples/tvhc/tvhc.h

```diff
@@ -20,11 +20,11 @@
 #endif  // __HELPBASE_H
 
 
-const int MAXSIZE = 80;
+const int MAXSIZE = 300;

```

Comment:
Although a path length of 80 is clearly not enough for legit cases, the actual issue here is the indiscriminate use of `strcpy`, which is a threat to life on Earth. Turbo Vision provides a safer `strnzcpy`. I'm willing to fix this myself.

Also, I'm not sure whether the `TProtectedStream::fileName` variable has any use at all. In any case, the length will never be greater than `MAXPATH` (defined in `<dir.h>`, may vary among platforms).